src/containers/__tests__/RoverList.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Cookies } from 'react-cookie';
+import configureStore from 'redux-mock-store';
+import RoverList from '../RoverList';
+import { fetchRovers } from '../../actions/rover';
+
+const cookiesValues = { auth_jwt: '1234' };
+const cookies = new Cookies(cookiesValues);
+
+describe('The RoverListContainer', () => {
+  const mockStore = configureStore();
+  let store;
+  let wrapper;
+  beforeEach(() => {
+    store = mockStore({
+      rover: {
+        isFetching: false,
+        rovers: [],
+      },
+    });
+    store.dispatch = jest.fn();
+    const context = { cookies };
+    wrapper = shallow(<RoverList store={store} />, { context });
+  });
+  test('dispatches an action to fetch rovers', () => {
+    wrapper.dive().props().fetchRovers();
```

Comment:
I think something here still needs to be mocked out because this is output to the log when running this test:
```
(node:2792) UnhandledPromiseRejectionWarning: SyntaxError: The string did not match the expected pattern.
    at XMLHttpRequest.open (/home/travis/build/rovercode/rovercode-ui/node_modules/jsdom/lib/jsdom/living/xmlhttprequest.js:482:15)
    at dispatchXhrRequest (/home/travis/build/rovercode/rovercode-ui/node_modules/axios/lib/adapters/xhr.js:45:13)
    at new Promise (<anonymous>)
    at xhrAdapter (/home/travis/build/rovercode/rovercode-ui/node_modules/axios/lib/adapters/xhr.js:12:10)
    at dispatchRequest (/home/travis/build/rovercode/rovercode-ui/node_modules/axios/lib/core/dispatchRequest.js:59:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
(node:2792) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 7)
(node:2792) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:2792) UnhandledPromiseRejectionWarning: SyntaxError: The string did not match the expected pattern.
    at XMLHttpRequest.open (/home/travis/build/rovercode/rovercode-ui/node_modules/jsdom/lib/jsdom/living/xmlhttprequest.js:482:15)
    at dispatchXhrRequest (/home/travis/build/rovercode/rovercode-ui/node_modules/axios/lib/adapters/xhr.js:45:13)
    at new Promise (<anonymous>)
    at xhrAdapter (/home/travis/build/rovercode/rovercode-ui/node_modules/axios/lib/adapters/xhr.js:12:10)
    at dispatchRequest (/home/travis/build/rovercode/rovercode-ui/node_modules/axios/lib/core/dispatchRequest.js:59:10)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
(node:2792) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 8)
```

Reply:
good catch here... I assumed it was an existing issue with a previously running spec.  just needed to mock axios in the RoverList container test.